sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -341,22 +390,63 @@ export class AssessmentController {
       const isReadyForCalculation =
         firstOutstandingStudentAssessment.id === job.variables.assessmentId;
       if (isReadyForCalculation) {
-        await this.studentAssessmentService.saveAssessmentCalculationStartDate(
-          assessmentId,
+        const saveAssessmentCalculationStartDate =
+          this.studentAssessmentService.saveAssessmentCalculationStartDate(
+            assessmentId,
+          );
+        const getProgramYearTotalAwards =
+          this.assessmentSequentialProcessingService.getProgramYearPreviousAwardsTotals(
+            assessmentId,
+            { alternativeReferenceDate: new Date() },
+          );
+        // Updates the calculation start date and get the program year totals in parallel.
+        const [, programYearTotalAwards] = await Promise.all([
+          saveAssessmentCalculationStartDate,
+          getProgramYearTotalAwards,
+        ]);
+        jobLogger.log(
+          `The assessment calculation order has been verified and the assessment id ${assessmentId} is ready to be processed.`,
         );
+        this.createOutputForProgramYearTotals(programYearTotalAwards, result);
+        result.isReadyForCalculation = true;
+        return job.complete(result);
```

Comment:
The result now will have the variables and values as the sum of awards for each award type and offering intensity, e.g., programYearTotalFullTimeCSGD and 2986 👍